CMakeLists.txt

```diff
@@ -8,7 +8,7 @@
 
 cmake_minimum_required(VERSION 3.16)
 
-project(iDynTree VERSION 2.0.2
+project(iDynTree VERSION 2.0.3
```

Comment:
Feel free to bump more (like 2.0.100), as 2.0.3 in the future could be be a patch release on the master branch.

Reply:
Done in https://github.com/robotology/idyntree/pull/802/commits/b10c4b8fbe6c016a574fac6c4ca1738680e85c7a